clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -4386,6 +4386,7 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
 
     return RValue::get(nullptr);
   }
+  case Builtin::BI__builtin_start_object_lifetime:
   case Builtin::BI__builtin_launder: {
```

Comment:
Should the `TypeRequiresBuiltinLaunder` function be generalized, at least in name?